api/MediaCapabilities.json

```
@@ -81,6 +81,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "configuration_keySystemConfiguration_parameter": {
+          "__compat": {
+            "description": "<code>configuration.keySystemConfiguration</code> parameter",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaCapabilities/decodingInfo#keysystemconfiguration",
+            "spec_url": "https://w3c.github.io/media-capabilities/#dom-mediadecodingconfiguration-keysystemconfiguration",
+            "support": {
+              "chrome": {
+                "version_added": false
```

I've put false, but perhaps this should be partial. I found https://chromestatus.com/feature/5765900795904000 which is clearly the precursor to the current spec. It isn't complete support for sure though.

---

See https://bugzilla.mozilla.org/show_bug.cgi?id=1898344

---

Modifying to "80" based on https://chromestatus.com/feature/5765900795904000 and response in https://bugzilla.mozilla.org/show_bug.cgi?id=1898344#c21

```suggestion
                "version_added": "80"
```